src/utils/problem.jl

```diff
@@ -108,7 +108,7 @@ Like `BasicResult`, but also returns the output reachable set given the input co
 """
 struct ReachabilityResult <: Result
 	status::Symbol
-	reachable::Vector{<:AbstractPolytope}
+	reachable::typejoin(Array{<:LazySet}, Tuple{<:LazySet})
```

Comment:
`typejoin(Array{<:LazySet}, Tuple{<:LazySet})` is `Any`. I don't mind making this field `Any` in principle, since it has no bearing at all on inference or performance, but I'm also not seeing where a `Tuple{<:LazySet}` is being returned. Was this change temporary during development?

Reply:
I had problems with the solver returning a Tuple but it seems that this is not longer needed!